client/pages/Assessment/pages/Section/Section.tsx

```diff
@@ -0,0 +1,53 @@
+import './section.scss'
+import React, { useEffect } from 'react'
+import { useCountryIso } from '@client/hooks'
+import { useAppDispatch } from '@client/store'
+import { SectionActions, useSectionTableData } from '@client/store/data/section'
+import { useParams } from 'react-router'
+import { AssessmentName } from '@meta/assessment'
+import { AssessmentActions, useSectionMetaData } from '@client/store/assessment'
+
+const Section: React.FC = () => {
+  const dispatch = useAppDispatch()
+  const countryIso = useCountryIso()
+  const sectionMetaData = useSectionMetaData()
+  const sectionTableData = useSectionTableData()
+  const { assessmentName, cycleName, section } =
+    useParams<{ assessmentName: AssessmentName; cycleName: string; section: string }>()
+
+  // Update section tables' metadata on countryIso(url) change
+  useEffect(() => {
+    dispatch(
+      AssessmentActions.getSectionTablesMetadata({
+        assessmentName,
+        cycleName,
+        section,
+        countryIso,
+      })
+    )
+  }, [countryIso])
+
+  // Update section tables' data on (countryIso ->) metadata change
+  useEffect(() => {
+    dispatch(
+      SectionActions.fetchSectionData({
```

Comment:
the names of the actions should be consistent

fetch or get. 
sectionMetadata and sectionData

Reply:
which is preferred?

Reply:
I saw we were using get in other actions